pgcontents/pgmanager.py

```diff
@@ -73,7 +73,11 @@ def _checkpoints_class_default(self):
         return PostgresCheckpoints
 
     def _checkpoints_kwargs_default(self):
-        kw = super(PostgresContentsManager, self)._checkpoints_kwargs_default()
+        try:
+            klass = PostgresContentsManager
+            kw = super(klass, self)._checkpoints_kwargs_default()
+        except AttributeError:
+            kw = {}
```

Comment:
It looks like this method changed names from `_checkpoints_kwargs_default` to `_default_checkpoints_kwargs` in https://github.com/jupyter/notebook/commit/a91af4d6420a3599df968a9452035de5a495af5c#diff-c604f4531aba170341bbf6e9a155d5daR124, which upgraded to the 4.x series of [traitlets](http://traitlets.readthedocs.io/en/stable/). The current defaults are: `{'parent': self, 'log': self.log}`.

Handling this gracefully across multiple Jupyter versions seems like a pain. I think the right way to handle this is probably to just drop support for the 3.x series of IPython/Jupyter (it's two major versions old now, so I don't feel too bad about losing support for it. We've also had https://github.com/quantopian/pgcontents/issues/32 open for a while with no objections raised.) and upgrade to the newer decorator-based traitlets APIs.

Dropping 3.x support is probably more work than we want to do in this PR, so until I get around to ripping out the old-style traitlets code, I think the right thing to do here is probably to hard code the defaults from upstream. Does that seem reasonable to you?

Reply:
alternatively i can just try to call the other method